tools/util_generate_hex_from_json.cpp

```diff
@@ -0,0 +1,148 @@
+//
+// Created by Eric Passmore on 1/14/23.
```

Comment:
Should check with someone on this. I don't know if this implies ownership license. Not sure ENF stance on adding a `Created by`.

Reply:
no problem I will remove, that was added by CLion.